product_docs/docs/epas/17/epas_security_guide/02_protecting_against_sql_injection_attacks/02_configuring_sql_protect.mdx

```diff
@@ -127,9 +127,9 @@ For each database that you want to protect, you must determine the roles you wan
 <!-- Consider replacing -->
```

Comment:
@tusharahuja, can you help with the new example here?

Reply:
The example is already there in the doc 

edb=# SELECT protect_role('appuser');
Output
 protect_role
--------------

(1 row)